helm/azure-operator-resource-chart/values.yaml

```diff
@@ -0,0 +1,15 @@
+clusterName: "test-cluster"
+commonDomain: "test-host-cluster.westeurope.azure.gigantic.io"
+commonDomainResourceGroup: "test-host-cluster"
+sshUser: "test-user"
+sshPublicKey: "ssh-rsa XXXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
We use this in aws-operator too but if it doesn't get overridden it causes problems.

As it's not valid base64 the cloudconfig fails. I think it's better to put a default key or just anything base64 here. So you can test the cluster comes up even if you can't SSH to the nodes.